api/v1alpha3/types.go

```diff
@@ -358,6 +358,11 @@ const (
 )
 
 // OSDisk defines the operating system disk for a VM.
+//
+// WARNING: this requires any updates to ManagedDisk to be manually converted. This is due to the odd issue with
+// conversion-gen where the warning message generated uses a relative directory import rather than the fully
+// qualified import when generating outside of the GOPATH.
+// +k8s:conversion-gen=false
```

Comment:
do we need `// +k8s:conversion-gen=false` so it doesn't generate `autoConvert_v1alpha3_OSDisk_To_v1alpha4_OSDisk` ?

Reply:
Yes, that is precisely what `+k8s:conversion-gen=false` does.